instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -37,18 +37,76 @@ def hello():
 Usage (Django)
 --------------
 
-Modify the application's ``wsgi.py`` file as shown below.
+Create a ``tracing.py`` file in your Django app as shown below.
 
 .. code-block:: python
 
+    from logging import getLogger
+    from pkg_resources import iter_entry_points
+
+    from opentelemetry import trace
+    from opentelemetry.sdk.resources import Resource
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.export import (
+        ConsoleSpanExporter,
+        BatchExportSpanProcessor,
+        SimpleExportSpanProcessor,
+    )
+
+    logger = getLogger(__file__)
+
+    def init_tracing():
+        tracer = TracerProvider()
+        trace.set_tracer_provider(tracer)
+
+        provider.add_span_processor(BatchExportSpanProcessor(ConsoleSpanExporter()))
+        auto_instrument()
+
+    def auto_instrument():
+      for entry_point in iter_entry_points("opentelemetry_instrumentor"):
```

Comment:
Why is this needed here? This seems to be doing the same as [`opentelemetry-instrument`](https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-instrumentation/setup.cfg#L52).

Reply:
It's needed for the tracing provider to evaluate multiple entry points